packages/bare-only/expo-notifications/.eslintrc.js

```diff
@@ -0,0 +1,2 @@
+// @generated by expo-module-scripts
```

Comment:
I had a problem with react-native-unimodules because it looks for directories with unimodule.json file inside and I wanted to add only expo-notifications to my example app. However, I found a better solution so I can now get rid of the bare-only directory.